app/controllers/locations_controller.rb

```diff
@@ -1,9 +1,13 @@
 class LocationsController < ApplicationController
   def index
-    @locations = Location.all.order(:name).page params[:page]
     if user_signed_in?
       @profile = current_user.profile
     end
+    if params[:search]
```

Comment:
This is fine, hound I guess doesn't understand that `if` logic can also just check for existence.